### PR TITLE
first write-up on creating GUPRIs

### LINKED ISSUE #382: Background knowledge on GUPRIs
- [x] identify author
- [x] write abstract
- [ ] agree with editors on abstract
- [ ] write recipe
- [ ] identify reviewer
- [ ] conduct review
- [ ] incorporate reviewer's comments
- [ ] publish recipe

[Background knowledge on Creating Resolvable Identifiers (1).docx](https://github.com/FAIRplus/the-fair-cookbook/files/7521540/Background.knowledge.on.Creating.Resolvable.Identifiers.1.docx)

### PR DESCRIPTION
**Pull request by robertgiessmann**
as tracked in #382 

needs probably more formal language? what do you think?

maybe this can be reworked by someone in the next bookdash?

would close #382

### CODE REVIEW

File: content/recipes/infrastructure/gupri.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+* HTTP
+* URL
+* DNS
+
+
+## Abstract
+
+Missing
+
```
Comment (tabbassidaloii) [inline content/recipes/infrastructure/gupri.md:26]: Adding the aim of this recipe:
The importance of Globally Unique, Persistent, Resolvable Identifier and introducing a few public services for minting GUPRIs.

File: content/recipes/infrastructure/gupri.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+Conclusion: Quick and easy solution; but not the most reliable. You have to register one purl "domain" to one user; if multiple users want to manage a common GUPRI space, they have to share the same credentials.
+
+
+### w3id.org 
+
+The service w3id.org emerged when purl.org was down. It is run by multiple stakeholders (see self-description on w3id.org) bound by a "social contract". 
+
+Management is more difficult than purl.org, because w3id.org does not have a graphical user interface / frontend, but relies on "raw" .htaccess files deposited on github.com. You need to have some knowledge about Apache .htaccess redirect rules, thus. A clear advantage is that it gives you more freedom in the implementation.
+
```
Comment (tabbassidaloii) [inline content/recipes/infrastructure/gupri.md:106]: I would add a note about the flags that can be used: https://httpd.apache.org/docs/current/en/rewrite/flags.html

File: content/recipes/infrastructure/gupri.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+````{panels_fairplus}
+:identifier_text: 
+:identifier_link: 
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 30
+:intended_audience: data_manager, data_scientist  
+:has_executable_code: nope
+:recipe_name: GUPRIs
+````
```
Comment (tabbassidaloii) [inline content/recipes/infrastructure/gupri.md:13]: :maturity_indicator: 1
:maturity_indicator: 1

**Review by tabbassidaloii — COMMENTED** [review]
Clearly written. Only added a few comments

**Comment by proccaserra** [comment]
@robertgiessmann @tabbassidaloii I will be closing this PR but opening a new one since migrating is totally out of synch with main.
you'll be tagged for another review. I have integrated all your comments + edited a few bit (abstract, conclusion, ref, formatting, toc)

**Pull request closed without merging**